src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListItem.tsx

```diff
@@ -92,3 +87,18 @@ export const ArtworkListItemFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+interface ArtworkListItemLinkProps extends RouterLinkProps {
+  isSelected: boolean
+}
+
+const ArtworkListItemLink = styled<ArtworkListItemLinkProps>(RouterLink)`
+  /* always */
+  border-radius: 10px;
+  margin-top: 2px; // otherwise top borders get cut off
+  display: block; // otherwise the child element collapses without a visible focus outline
+
+  /* when this list is currently being viewed (isSelected) */
+  border: solid 1px ${props => (!!props.isSelected ? "black" : "transparent")};
```

Comment:
**question**: did you clarified black border color with Ryan?

Reply:
if not, what do you think about using `outline-offset: 0` here? It will look something like this


https://github.com/artsy/force/assets/3513494/0f660997-86a9-4954-84e6-d6c8418e2594